### ISSUE TITLE
ERROR: STEP14 -> BINNING & STEP 18 checkM_batch.pl

### ISSUE THREAD

**Issue by sogada** [jtamames/SqueezeMeta#716]
Hello, 
I've tried running the pipeline and I'm getting the following errors as shown in the attached image. I have also attached the syslog file and one .tax file. 
I ran the test_install.pl script and all checks were successful. 
I'm running the pipeline on a server and installed it via Conda as instructed here. What could be the issue?

[syslog.zip](https://github.com/jtamames/SqueezeMeta/files/12163481/syslog.zip)
![squeezemeta](https://github.com/jtamames/SqueezeMeta/assets/47321832/1e713108-d01c-4279-a42d-d34a85de23e3)

**Comment by jtamames** [comment]
Hello!
Your dataset is rather small and as a result, the bins are very low in size. All of them are skipped because of low length, and then the pipeline stops because of not finding any suitable bins. We will remove this check in upcoming versions, because it makes the pipeline to stop even if no error was produced. In the meantime, just restart the project via --restart -step 18 to finish the run.
Best,
J

**Comment by fpusan** [comment]
What is your OS and version?

The error in step 14 seems similar to the one described in #692. In short, recompiling `metabat2` and `jgi-summarize-contigs-depth` from source may fix it for you. I need to find a more permanent fix, but it is difficult to distribute binaries that work in all distributions and versions.

This error means that metabat2 will fail, but the other binning methods (by also CONCOCT, but you can also add MaxBin to the mix) will provide binning results so the run does not fail.

However you then have a second issue in which your bins are too short.

Is this happening with the test dataset that we provide when downloading the databases?

Otherwise as Javier says it maybe due to your data not producing suitable bins.

**Comment by jtamames** [comment]
Oh true, I didn't notice metabat2 failing. Sorry about that

**Comment by sogada** [comment]
Hello jtamames and fpusan, 
Thank you for your fast response. It's highly appreciated and commendable. I run Ubuntu 18 LTS. I have to admit, though, I used a small dataset for my test run. Let me run with more samples as suggested above and see what I'll get. I'll notify you once my run is complete. I will also try using the test data to rule out any data-related issues.

**Comment by sogada** [comment]
Hello jtamames and fpusan,
I ran your test data and later my whole dataset. You were both right. 

1. The first issue was my small dataset. After running with my whole dataset, the pipeline ran to completion.
2. Seems there's an issue with metabat2 failing in my machine (see attached photo). I got the same error with both my data and your test data. I think I can use concot and maxbin as alternatives. Does metabat2 offer any significant advantages over the others?
![github](https://github.com/jtamames/SqueezeMeta/assets/47321832/9e9f2107-1dc3-4b6b-bb68-b978b3989a9d)
